lib/releaselib.sh

```diff
@@ -673,9 +673,9 @@ release::gcs::locally_stage_release_artifacts() {
 
   # Upload the "naked" binaries to GCS.  This is useful for install scripts that
   # download the binaries directly and don't need tars.
-  mapfile -t platforms < <(
+  mapfile -d ' ' -t platforms < <(
     cd "$release_stage/client" || return 1
-    echo -- *)
+    echo ""*)
```

Comment:
this feels wrong 🤔

Reply:
should be `echo ./*` ?

Reply:
or we need no leading ./? in which case see my previous robust solution for this:
https://github.com/kubernetes/kubernetes/blob/4b3987d0c390fbac65df5c0a1cf7f89d6ddfc6ce/hack/lib/util.sh#L543

https://github.com/kubernetes/kubernetes/blob/4b3987d0c390fbac65df5c0a1cf7f89d6ddfc6ce/hack/make-rules/make-help.sh#L26

Reply:
Done!